README.md

```diff
@@ -1,54 +1,91 @@
 
-Let's make this rendering process of choosing a view a little more explicit using the rails render method (equiv to sinatra's erb method if you're familair with it).
+If you refresh the browser now you will see a new error: ```The action 'about' could not be found for StaticController```
 
-Render can accept a string that points to a view template relative to 'views', so we can use render 'static/about'. If you think about it, that's a great place to put a view that relates to the about action in the static controller. In fact, it's such a reasonable ad convineinent place to put it that rails actually assumes, given no other render instructions, that's where you want to put it. This is a big principal in rails, convention over configuration. That's why our action worked without a render call. As a beginner we believe being explicit is easier at first then being implicit, learn the conventions and then use them.
+We're making good progress (even though we're using EDD - error driven development), and it's good to see each of the errors so that when you encounter these in your real world projects you will know how to fix them. This current error is fixed by adding the following method in the static controller:
 
-And like that we've created our first rails request - it's a static request, no data is being used, no matter what, everytime we load our about page it will be exactly the same. there is nothing dynamic about it. But that teaches us the general methodology of responding to a request with rails.
+```ruby
+def about
+end
+```
 
-draw a route.
-map it to a controller action.
-program the action
-create a template to render
-render that template.
+Hitting refresh in the browser will give you a 'Template is missing' error, specifically it says: ```Missing template static/about...```
+
+We're very close to getting our view to show up, but before we make our final addition it's important to understand the difference between explicit and implicit rendering for the views. Rails gives us two options for how views are mapped between the controller and view files:
+
+* **Explicit rendering** - for explicit rendering, Rails lets you dictate what view file that you want to have the controller action mapped to.
+* **Implicit rendering** - for implicit rendering, Rails follows a standard convention that automatically looks for the view file with the same name as the controller action.
+
+First let's try out explicit rendering, create a new directory in the view's directory called ```static``` and create a new file called ```some_page.html.erb```. In that file add some basic HTML code, such as:
+
+```html
+<h1>Hello from some page</h1>
+```
+
+Inside the ```about``` method in the controller add the following code:
+
+```render "static/some_page"``` you can also use: ```render "some_page"``` (Rails will automatically look inside the view directory with the same name as the controller, but you can also give the full view path).
```

Comment:
Also just to give them a bit of context I'd just put the full code snippet.

Modify your `about` method to look like this:

```
def about
  render "static/some_page"
end
```